### PR TITLE
[WiP] refactors: real usage in tests and docs

### PR DESCRIPTION
**Pull request by arnoFleming**
## Proposed changes

Make the test (and docs) more in line with how devs like to work

## Types of changes

What types of changes does your code introduce to wetransfer_ruby_sdk?

- [X] Docs (missing or updated docs)
- [ ] Bugfix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [x] Improvement / maintenance (removing technical debt, performance, tooling, etc...)

## Checklist

_Put an `x` in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your code._

- [ ] I have read the [CONTRIBUTING](https://github.com/WeTransfer/wetransfer_ruby_sdk/blob/master/.github/CONTRIBUTING.md) doc
- [ ] I have added necessary documentation (if appropriate)
- [ ] Code changes are covered by unit tests.
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: spec/board_integration_spec.rb [hunk 1]
```diff
@@ -1,19 +1,8 @@
...
-      File.open(TWO_CHUNKS_FILE_NAME, 'w') do |f|
-        f.puts('-' * (PART_SIZE + 3))
-        puts File.absolute_path(f)
-      end
-    end
-  end
-
-  let(:two_chunks) { File.open("#{Dir.pwd}/#{TWO_CHUNKS_FILE_NAME}", 'r') }
+  let(:small_file_path) { 'spec/files/Japan-02.jpg' }
+  let(:big_file) { File.open('spec/files/Japan-01.jpg', 'rb') }
```
Comment (arnoFleming) [inline spec/board_integration_spec.rb]: @bermannoah , @davidenko87 should we replace the `'rb'` flag with just `'r'`? the binary flag is only on Windows necessary.

File: Guardfile [hunk 2]
```diff
@@ -10,4 +10,4 @@ group :red_green_refactor, halt_on_fail: true do
   end
 end
 
-clearing :on
```
Comment (arnoFleming) [inline Guardfile]: this kinda breaks my flow. @davidenko87 could we do without this? or does removing this break your flow?

Comment (davidenko87) [reply Guardfile]: i can live without this

File: spec/board_integration_spec.rb (hunk 1 above)
Comment (davidenko87) [reply spec/board_integration_spec.rb]: 👍

**Comment by arnoFleming** [comment]
This is closed, since it is squashed/merged in #38

**Pull request closed without merging**